pkg/exchange/binance/exchange.go

```diff
@@ -24,6 +24,8 @@ var log = logrus.WithFields(logrus.Fields{
 	"exchange": "binance",
 })
 
+var binanceStartDate = time.Date(2017, 8, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
we can reuse the launchTime from the binance exchange

Reply:
Is launch time reasonable? I set this because it was about the time when binance was founded.

Reply:
yeah, I mean I have declared the same var, maybe we should use it

Reply:
Done. Plz check